modules/messages/client/components/ThreadReply.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import TrEditor from '@/modules/core/client/components/TrEditor';
+import plainTextLength from '@/modules/core/client/filters/plain-text-length.client.filter';
+
+export default function ThreadReply({ onSend, cacheKey }) {
+  const { t } = useTranslation('messages');
+
+  const [editorKeyCounter, setEditorKeyCounter] = useState(0);
+  const [content, setContent] = useState(() => getDraft() || '');
+
+  function send(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    const value = content.replace(/<br><\/p>$/, '</p>'); // remove the last <br>
```

Comment:
We could switch this to server-side too but ok here as well.

Reply:
I did the opposite :) - moved it inside TrEditor (I think it should be a feature of medium editor, so it emits better html in the first place.

Reply:
Hehe, I'm not opposed to that idea either. I guess both are equally funky and not fixing the root cause anyway. :-)